portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function guessMyTrip()
+{
+  const places=["thailand", "singapore", "tokyo", "australia","peru", "hawaii", "vietnam"];
+  var input =document.getElementById("user_input").value;
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
I will change var to let now